### DISCUSSION TITLE
Publishing plugins to GitHub packages

### DISCUSSION THREAD

**Discussion by stringbean** [sbt/sbt#7162] [Q&A]
Has anyone managed to publish an sbt plugin to GitHub packages? We've tried both plain `publishTo` settings (that work for our normal projects) and the sbt-github-packages plugin but attempting to use the published plugin fails to resolve.

I remember when GH packages was first released there were issues with the package naming used for Scala dependencies. Do we need to raise this with GH?

**Comment by eed3si9n** [comment]
I think the problem is that GitHub package doesn't expose publicly accessible static site, and requires authentication via token.
https://github.com/djspiewak/sbt-github-packages implements resolver for that, but I have not personally tried it.

**Comment by reid-spencer** [comment]
It worked for me, today.  That is, I used sbt-github-packages to publish an sbt plugin to my GitHub Packages. 

However, once there, it seems that I cannot convince a GitHub workflow to resolve it outside of development.  In a GitHub CI (workflow) environment, I consistently get "unauthorized" (not "notfound") from the  resolver.  I've been debugging it for hours with Claude.ai help. Everything we've tried failed. The repository is only useful if READING it is permitted too :)
